wordpress-mobile/WordPress-Android
# Refactor the layout of notifications item

Part of #20024

This PR contains:
1. Correct the height of the layout.
2. Reduce a nested layout layer for better performance (replace with ConstraintLayout).

-----

## To Test:
1. Open the Jetpack app and switch to the notifications tab.
2. The UI of notifications should match the new design. (See: yWt5gg3nWORhu079Qfv3mS-fi-51_1131)
3. Done, thank you!


-----

## Regression Notes

1. Potential unintended areas of impact

    - Notifications tab

3. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manuel

4. What automated tests I added (or what prevented me from doing so)

    - None

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
